Modules/posixmodule.c

```diff
@@ -5349,7 +5359,20 @@ os_posix_spawn_impl(PyObject *module, path_t *path, PyObject *argv,
     }
 
     if (file_actions != Py_None) {
-        if (parse_file_actions(file_actions, &file_actions_buf)) {
+
+        /* There is a bug in old versions of glibc that makes some of the
```

Comment:
There are two large comments about this issue which are not duplicates of one other. It would be better to merge them.

Reply:
Done in commit 80918db